Build community portal landing page

petesfrench opened the pull request:
## Done

- Build community landing page, based on the [copydoc](https://docs.google.com/document/d/170WJDDZmmoMAuSlfQon5EvZl_qpUQEVw_r3cQNAP3Bs/edit?tab=t.0) and [figma](https://www.figma.com/design/NtIK22CtDUTO8iQWoaUWlq/ubuntu.com-community?node-id=149-5021&m=dev) 
- Setup up connection with Discourse
- Add setting for 3 items in latest-news patern
- Create u-clamp class for limiting text to two lines

## QA

- Open the [demo](https://ubuntu-com-15273.demos.haus/community)
- Compare against the [copydoc](https://docs.google.com/document/d/170WJDDZmmoMAuSlfQon5EvZl_qpUQEVw_r3cQNAP3Bs/edit?tab=t.0) and [figma](https://www.figma.com/design/NtIK22CtDUTO8iQWoaUWlq/ubuntu.com-community?node-id=149-5021&m=dev) 

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-17323

## Screenshots

![image](https://github.com/user-attachments/assets/932df525-988c-4f31-8854-213cb1cc2276)

mattea-turic commented:
Thank you @petesfrench for your work on this! And thanks for uploading the assets!

It already looks great, so literally just one comment:
- [ ] For the first section ("Ubuntu Summit"), could we add in the "Highlighted event" subtext – I hid it temporarily and forgot to make it visible again
<img width="500" alt="Screenshot 2025-06-26 at 18 12 04" src="https://github.com/user-attachments/assets/e8c72b60-7e67-4bf8-ae1e-6cb47e9f0b79" />

petesfrench commented:
> Thanks @petesfrench, this looks so good!! One q - Aaron said he had some images for the LoCos and I saw your comment on the images being taken from Discourse. Does that mean that instead of having to upload the image onto asset manager for us to update, he can do so on Discourse now and in the future and it'll update on the page?
Exactly. The locos images will be stored in the asset manager and then uploaded to discourse, finally being displayed on the page. This will all be handled by the community team and wont need interjection from us.

> 
> * [x]  Aaron edited the text for the highlighted event - could you please use the new copy, thank you!
Done, thanks!
> * [x]  Event headings truncation - if an event heading exceeds 2 lines, will the text be truncated? Mattea left a comment on Figma but we didn't annotate it. Just checking if this was implemented
Yes, the event titles truncate after two lines
> * [x]  [View all newsletter issues] link doesn't exist. I assume we can link to Discourse https://discourse.ubuntu.com/c/uwn/124 but I tagged Aaron in a comment on the copydoc.
This page will exist at one point in the soon future
> * [x]  "[Take a look at how our community can support you.](https://ubuntu-com-15273.demos.haus/community/support)" link is invalid but the page exists on the live site (https://ubuntu.com/community/support). Is this because we moved the existing pages to u.com/community/docs? If so should we have this link to /community/docs/support instead?
I have updated it to link to /community/docs/support  as suggested
